purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,260 @@
+package com.revenuecat.purchases.customercenter
+
+import com.revenuecat.purchases.paywalls.PaywallColor
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+typealias RCColor = PaywallColor
```

Comment:
Not a big fun of these typealiases, but it's how we have it in iOS, and it kinda made sense now.